docs/src/operations/requirements.md

```diff
@@ -55,7 +56,7 @@ made.
 - CPU
   - 16 cores / 32 threads, or more
 - RAM
-  - 512 GB or more if `account-index` is used
+  - 512 GB or more if `account-index` is used (1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
```

Comment:
This is a good addition, especially with link to the other page. Just a tweak or two:
> - 512 GB or more if an [account-index](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing) is used, 1 TB+ for all three

Reply:
Fixed wording in d7a3a7aa8e